### DISCUSSION TITLE
Need Help: How to stop showing the pnpm update.

### DISCUSSION THREAD

**Discussion by phanimurari** [pnpm/pnpm#5485] [Q&A]
![image](https://user-images.githubusercontent.com/45779787/195555243-2ebc8092-1270-4188-9c41-012dbd2b152f.png)

**Comment by lvqq** [comment]
Pnpm do have provided a option `update-notifier`, but it was added in v6.29.0: https://pnpm.io/6.x/npmrc#update-notifier

**Reply by phanimurari** [reply]
Thanks @lvqq 

Will there be any issue if i update my current version from 5.17.1 to 6.32.10 ?

**Reply by zkochan** [reply]
why do you use v5 if v7 is the latest?
